gradle.properties

```diff
@@ -1,14 +1,27 @@
 # Done to increase the memory available to gradle.
 org.gradle.jvmargs=-Xmx1G
 
-enigma_version=1.2.1
-enigma_plugin_version=1.2.1
-stitch_version=0.6.1
+# Fabric & Quilt Libraries
+enigma=1.2.1
+enigma_plugin=1.2.1
+fabric_loader=0.14.10
+findbugs=3.0.2
```

Comment:
Two things: this isn't a fabric/quilt dependency, and it's name should be jsr305